### PR TITLE
PP-5115: Add Dockerfile and startup scripts

### PR DESCRIPTION
**Pull request by rjbaker**
### WHAT
Adds a simple openjdk11 alpine container, startup script and local build script similar to other Pay application services.

### HOW
This will allow a local application container to be built and run (requires local installation of maven). Once merged, further build pipeline development may continue.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+RUN wget -qO - https://s3.amazonaws.com/rds-downloads/rds-ca-2015-root.pem       | keytool -importcert -noprompt -cacerts -storepass changeit -alias rds-ca-2015-root \
+ && wget -qO - https://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem | keytool -importcert -noprompt -cacerts -storepass changeit -alias rds-combined-ca-bundle
+
+RUN ["apk", "add", "--no-cache", "bash"]
+
+ENV PORT 8080
+ENV ADMIN_PORT 8081
+
+EXPOSE 8080
+EXPOSE 8081
```
Comment (rjbaker) [inline Dockerfile]: Is the admin port required by this application?

Comment (mrlumbu) [reply Dockerfile]: No

File: docker-startup.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  i=0
+  truststore_pass=changeit
+  for cert in "$CERTS_PATH"/*; do
+    [ -f "$cert" ] || continue
+    echo "Adding $cert to default truststore"
+    keytool -importcert -noprompt -cacerts -storepass "$truststore_pass" -file "$cert" -alias custom$((i++))
+  done
+fi
+
+java $JAVA_OPTS -jar *-allinone.jar waitOnDependencies *.yaml
```
Comment (mrlumbu) [inline docker-startup.sh]: we don't have this yet, and maybe this isn't the best way to do it. Could maybe just use db status check provided by dropwizard? https://www.dropwizard.io/0.7.1/docs/manual/migrations.html

Comment (rjbaker) [reply docker-startup.sh]: I'll comment this out for now, but feel free to amend as you continue development. This should be enough container setup to get the Jenkins build pipeline running.

Comment (mrlumbu) [reply docker-startup.sh]: 👍

**Pull request merged**